src/datascience-ui/renderers/render.tsx

```diff
@@ -87,3 +91,7 @@ export class CellOutput extends React.Component<ICellOutputProps> {
         return this.renderOutput(data);
     }
 }
+
+function isAltairPlot(mimeType: string) {
+    return mimeType.includes('application/vnd.vegalite.v4+json');
```

Comment:
I think it would be anything with 'vega' in it. There can be many different versions.

Reply:
It looks like there are two possible values:
- application/vnd.vega.v4+json
- application/vnd.vegalite.v4+json

where the version number can change. So would something like this work?

```suggestion
    return mimeType.includes('application/vnd/vega');
```

Reply:
@joyceerhl your suggestions looks good to me. Should catch the types we want without casting too wide a net.

Reply:
Sounds good to me.